dapps/evancore.vue.libs/src/components/login/login.ts

```diff
@@ -18,37 +18,40 @@
 */
 
 // vue imports
-import Vue from 'vue';
 import Component, { mixins } from 'vue-class-component';
 import { Prop } from 'vue-property-decorator';
 
 // evan.network imports
 import EvanComponent from '../../component';
 import * as bcc from '@evan.network/api-blockchain-core';
 import * as dappBrowser from '@evan.network/ui-dapp-browser';
+import { getDomainName } from '../../utils';
 
 /**
  * Handles the password input of a user and checks, if it's correct and it's profile can be
  * encrypted with that password. Used by the dapp-wrapper to login the current user if needed. Will
  * send an `logged-in` event including the users provided password.
  *
+ * TODO: Rethink the naming. We have sign-in, sign-up, signed-in and login (this)
```

Comment:
What would be your suggestion?

Current functionality of the components:
- sign-up: registering a new profile on evan
- sign-in: recovering a already existing profile using a mnemonic on evan
- signed-in: user has signed-in / signed-up and was opening a e-mail invitation of a other user
- login: unlock current logged in profile with password

Reply:
The refactoring removed the sign-in component already. 
The comment is only about the naming of the other components. This is simply a reminder for the later bigger refactoring.